packages/server/src/controllers/files.ts

```diff
@@ -17,8 +23,19 @@ const dummy_file = {
 
 export default class FilesController {
   static create: Middleware = async (ctx) => {
-    FilesValidator.validateForCreate(ctx)
-    await File.create('test')
+    const { body: { owner }, file } = ctx.request
+    try {
+      FilesValidator.validateForCreate(ctx)
+      for await (const ipfsFile of ipfs.add(globSource(file.path))) {
+        dummy_file.cid = ipfsFile.cid.toString()
```

Comment:
Idea: Curious about what other information is being returned in the `ipfsFile` object, maybe it worths caching part of it

Reply:
Adding some stuff to the database in #4